dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/SubSnpCoreFieldsToVariantProcessor.java

```diff
@@ -49,40 +53,57 @@
 
     private final String dbsnpBuild;
 
+    private VariantFrequenciesBuilder frequenciesBuilder;
+
     public SubSnpCoreFieldsToVariantProcessor(int dbsnpBuild) {
         this.dbsnpBuild = String.valueOf(dbsnpBuild);
+        frequenciesBuilder = new VariantFrequenciesBuilder(new FrequenciesInfoParser());
     }
 
     @Override
     public Variant process(SubSnpCoreFields subSnpCoreFields) throws Exception {
-        if (areGenotypesInvalid(subSnpCoreFields)) {
+        if (areGenotypesAndFrequenciesInvalid(subSnpCoreFields)) {
```

Comment:
Maybe we could just have a double condition? Each of the methods has a clear name, so it seems a bit redundant to create a wrapper.